features/util.go

```diff
@@ -2,12 +2,15 @@ package features
 
 import (
 	"fmt"
-	configv1 "github.com/openshift/api/config/v1"
 	"net/url"
 	"strings"
+
+	configv1 "github.com/openshift/api/config/v1"
 )
 
 // FeatureGateDescription is a golang-only interface used to contains details for a feature gate.
+//
+//nolint:all
```

Comment:
Did this work? The linter shouldn't really be linting this file or even maybe package probably, might have to exclude at the `.golangci.yaml` level

Reply:
yup added top level instead